### ISSUE TITLE
about the SRRS in desnowing task

### ISSUE THREAD

**Issue by chensming** [c-yn/SFNet#9]
Hello, when I finished downloading the SRRS dataset, I found two problems, 
1）the training collection dataset has overlapping images， such as the red box in the image

![image](https://github.com/c-yn/SFNet/assets/45591767/e79a112d-ccab-426f-b7ae-7358508b8e69)

2) The training and test sets have duplicate images, and the training set is shown in the green box above. A similar situation exists with test sets such as 302.jpg and 303.jpg

![image](https://github.com/c-yn/SFNet/assets/45591767/23d22ab6-287c-40c5-a41a-af5c02a40284)

I hope you can answer my questions, thanks in advance!

**Comment by c-yn** [comment]
Thanks for your question.

This is because different snow patterns correspond to the same gt (background).

**Comment by chensming** [comment]
Thanks for your reply.

Yeah you're right. But another question is, it seems like the authors didn't take this into consideration when dividing the training and test sets, which then leads to the same scenarios in the training and test sets, e.g. 0.jpg in the training set and 1.jpg in the test set have the same scenarios, which seems like it would lead to high metrics for the model?

**Comment by chensming** [comment]
Also, I noticed that the CSD dataset downloaded from the [link](https://drive.google.com/file/d/1pns-7uWy-0SamxjA40qOCkkhSu7o7ULb/view?usp=sharing) you provided on readme.md has an overlap between the training set and the test set.
![image](https://github.com/c-yn/SFNet/assets/45591767/22cdd5cc-6f3a-4648-9fb5-81afe6737596)

**Comment by c-yn** [comment]
Hi, CSD shares the same gt/background between training and testing sets.

**Comment by chensming** [comment]
> Hi, CSD shares the same gt/background between training and testing sets.

ok. Thanks.
